README.md

```diff
@@ -1,2 +1,92 @@
 # Hello_world
 First proyect of microverse course 
+<a name="readme-top"></a>
+
+
+  <h3><b>Microverse README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> This is the first project of microverse course, it's help me to get familiar with github, linters, and gitflow
+
+**[Hello Microverse]** is a... proyect to master git and learn to set up linters 
+
+## 🛠 Built With <a name="built-with">HTML5&CSS3</a>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The proyect has this features.
+
+- **[gitflow]** To get the proper flow of work 
+- **[git]** To control the versions
+- **[linters]** To get the errors
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://htmlpreview.github.io/?https://github.com/duducus/Hello_world/blob/feature/index.html)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone yourlink repo [https://github.com/duducus/Hello_world]
+
+cd your folder name
+```
+
+### Install
+
+Install this project with:
+
+```
+npm install
+```
+
```

Comment:
- [ ] Please don't remove the following sections:

- Getting Started
- Usage
- Run tests
- Deployment
 
- Check below :point_down:  my suggestions :slightly_smiling_face: 
```suggestion
## 💻 Getting Started <a name="getting-started"></a>

To get a local copy up and running, follow these steps.

### Prerequisites

In order to run this project you need:

- You need to have NodeJS installed
- A Web Browser (Google Chrome, Firefox, etc)
- A Code Editor (Notepad++, VSCode, etc)

### Setup

Clone this repository to your desired folder:

```
git clone yourlink repo https://github.com/duducus/Hello_world
cd your folder name
```

### Install

Install this project with:

```
npm install
```

### Usage

To run the project, execute the following command:

Double-click on the index file (index.html)

### Run tests

To run tests, run the following command:

N/A

### Deployment

You can deploy this project using:

Deploy this project on any websever

<p align="right">(<a href="#readme-top">back to top</a>)</p>
```